test/test_mix.py

```diff
@@ -23,17 +23,17 @@ async def test_in_interface(self):
         series = mix.Series(io1, io2)
         assert await series.process(["Hello"]) == ["Hello World!"]
 
-    @pytest.mark.asyncio
-    @pytest.mark.flaky
-    async def test_with_external(self):
-        io1 = gr.Interface.load("spaces/abidlabs/image-identity")
-        io2 = gr.Interface.load("spaces/abidlabs/image-classifier")
-        series = mix.Series(io1, io2)
-        try:
-            output = series("gradio/test_data/lion.jpg")
-            assert output["lion"] > 0.5
-        except TooManyRequestsError:
-            pass
+    # @pytest.mark.asyncio
+    # @pytest.mark.flaky
+    # async def test_with_external(self):
```

Comment:
How come this test was failing? And how come #1967 will fix it if it has nothing to do with sync vs async?

Reply:
So this test is failing because the serialization/deserialization logic is not correct anymore in `Series`. 

As part of #1967, I'm actually rewriting the serialization logic, so this will either be fixed automatically or I'll take another look at it and fix it. 

We felt it would be unnecessary effort to fix it only to change the way we do serialization.

Reply:
Fair point! We should try to always keep the ci on main passing though. What if someone decides to cut a release tomorrow (to test the new queue for example) and they're not aware that some of the code is broken on main?

Reply:
Ah I agree! But if you guys think it's all right, we can make an exception in this case, because I think it's quite unlikely a user will run into this edge case (it only happens when you use `Series` with `external`) in the time it takes for us to fix it.

Reply:
Yeah, being flexible is alright. We don't have a upcoming stable release soon as well.